alvr/graphics/src/stream.rs

```diff
@@ -38,7 +36,8 @@ const _: () = assert!(
 pub struct StreamViewParams {
     pub swapchain_index: u32,
     pub reprojection_rotation: Quat,
-    pub fov: Fov,
+    pub input_view_params: ViewParams,
+    pub output_view_params: ViewParams,
```

Comment:
If you use input/output view params, then reprojection_rotation has no purpose anymore

Reply:
Yeah I was humming and hawing about removing that, it's redundant but also I hope it doesn't cause too much of an issue with C API users. I'll go ahead and remove it though.

Reply:
Resolved, removed it and double checked on the math so the C API can remain unchanged until someone wants full poses there.